ELLogTests/ELLogTests.swift

```diff
@@ -43,29 +23,31 @@ class ELLogTests: XCTestCase {
         super.tearDown()
     }
     
-    func testExample() {
-        // This is an example of a functional test case.
-        //XCTAssert(true, "Pass")
-
+    
+    func runTestsOn(logger: Logger) {
         let number = NSNumber(integer: 1234)
-        let _: TestEnum = .One
-
-        CLSLogv("hello = %@", getVaList([number]))
-
-        //Logger.defaultInstance.removeAllDestionations()
-
-        Logger.defaultInstance.log([.Debug, .Info], message: "hello \(number)")
+        
+        logger.removeAllDestionations()
```

Comment:
`removeAllDestionations()`, heh looks like we've had a typo in this method name for awhile. mind correcting it as part of the breaking change?